### PR TITLE
Maven central

### PR DESCRIPTION
**Pull request by mvsoder**
Create scripts to release the artifacts and to generate the artifact bundle for Maven Central deployment

### CODE REVIEW

File: subprojects/testfx-core/testfx-core.gradle [hunk 1]
```diff
@@ -16,7 +16,8 @@
  */
 apply plugin: 'java-library'
 
-ext.pomDescription = 'TestFX Core'
+ext.pomName = 'TestFX Core'
+ext.pomDescription = 'TestFX core library'
 ext.moduleName = 'org.testfx'
 ext.openjfxVersion = '12'
```
Comment (svenruppert) [inline subprojects/testfx-core/testfx-core.gradle:22]: here is the version 12 in the other modules the version is 11 - why?

**Review by svenruppert — COMMENTED** [review]
1) - see comment, please.
2) - add both of us to the developer sections, please.

File: subprojects/testfx-core/testfx-core.gradle (hunk 1 above)
Comment (mvsoder) [reply subprojects/testfx-core/testfx-core.gradle:22]: Probably just oversight. I will set this to 11 as the others are.

**Comment by mvsoder** [comment]
Updates are complete

**Review by svenruppert — APPROVED** [review]
LGTM

**Pull request merged**